src/ec/packages/ec-component-hero-banner/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ecl-twig/ec-component-hero-banner",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Hero Banner",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@ecl-twig/ec-component-button": "^0.0.1-alpha"
+  },
+  "devDependencies": {
+    "@ecl/ec-resources-icons": "~2.1.0",
+    "@ecl/ec-specs-hero-banner": "~2.1.0",
```

Comment:
please use the same version for all ECL packages (`^2.1.1`)

Reply:
Updated in https://github.com/ec-europa/ecl-twig/pull/49#discussion_r262992520